src/features/serviceProxy/index.ts

```diff
@@ -15,8 +15,12 @@ export default function init(stores: {
 }) {
   debug('Initializing `serviceProxy` feature');
 
+  const setIsEnabled = action((value: boolean) => {
```

Comment:
Is there a reason for the different style of the code? All other places its `@action <funcName>`, while here its different?

Reply:
I tried that and typescript didn't like it. I'm guessing `@action` only works inside a class - which is why I followed that approach.

Reply:
ok - resolving my comment. LGTM!